flink-java/src/main/java/org/apache/flink/api/java/sampling/BernoulliSampler.java

```diff
@@ -28,12 +28,16 @@
  * Bernoulli experiment.
  *
  * @param <T> The type of sample.
+ * @see <a href="http://erikerlandson.github.io/blog/2014/09/11/faster-random-samples-with-gap-sampling/">Gap Sampling</a>
  */
 public class BernoulliSampler<T> extends RandomSampler<T> {
 	
 	private final double fraction;
 	private final Random random;
 	
+	//THRESHOLD	 is a tuning parameter for choosing sampling method according to the fraction
```

Comment:
Format: A blank after // and no extra blanks after THRESHOULD, and a period at the stop.